rust_gmf_bench/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ghcr.io/rust-lang/rust:nightly-bookworm
```

Comment:
Can we use the same image as the rest of Rust implementations?

Reply:
I've updated the MR to use the same Rust image as other implementations